service/lsp/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include_directories(include)
```

Comment:
It is necessary if we want to refer to `lspservice/lsp_types.h` and `lspservice/lspservice.h` from their .cpp files under src.

(It could still be done without this statement, but since all other subdirectories used this approach, I would recommend keeping it this way)

Reply:
Noted, it can then remain like this.